scripts/min_glibc_version.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# This scripts determines the minimum GLIBC version required for a dynamic library.
+
+set -e -o pipefail
+
+# List the dynamic symbols of the library
+# Note: `ldd` does not work reliably when inspecting cross-compiled ARM binaries on x86_64
```

Comment:
> when inspecting cross-compiled ARM binaries on x86_64

Q: Would we ever need to run this script for `ARM` binaries?

Reply:
We build for a suite of targets, including `aarch64-unknown-linux-gnu` and `x86_64-unknown-linux-gnu`, which are listed under the `napi.triples.additional`.

We need to ensure that all of our GNU targets satisfy the target GLIBC version, so we call this script from the CI host (ubuntu 22.04, x86_64) and inspect with it both of the targets, including the `aarch64` (ARM64); that's what this PR introduces.

Reply:
Thanks for explaining.